vignettes/monaLisa.Rmd

```diff
@@ -448,24 +451,21 @@ colnames(se)[se$selected]
+The stability paths visualize how predictors get selected, decreasing regularization stringency (from left to right): 
 ```{r plotStabilityPaths}
 plotStabilityPaths(se)
 ```
-
 Each line corresponds to a predictor, and we can see the selection probabilities as a function of the regularization steps, corresponding to decreasing values for the lambda regularization parameter in lasso. The predictor (TF motif) selection happens at the last step, given the specified minimum probability.
 
-We can also visualize the selection probabilities of the selected TF motifs, optionally multiplied by the sign of the correlation to the response vector, to know how the TF relates to the change of accessibility (`directional` parameter). Note that although one can vary the `selProbMinPlot` parameter which sets the selection probability cutoff, it is best to re-run randomized lasso stability selection with the new cutoff, as this influences the other parameters model uses internally. See @StabSel for more details.
+\ 
 
-```{r plotSelProbs, fig.width=15, fig.height=7}
+We can also visualize the selection probabilities of the selected TF motifs, optionally multiplied by the sign of the correlation to the response vector, to know how the TF relates to the change of accessibility (`directional` parameter). Note that although one can vary the `selProbMinPlot` parameter which sets the selection probability cutoff, it is best to re-run randomized lasso stability selection with the new cutoff, as this influences other parameter choices the model uses internally. See @StabSel for more details.
+```{r plotSelProbs, fig.width=15, fig.height=9}
 plotSelectionProb(se, directional = TRUE)
 ```
 
-Next, we visualize the correlation structure of the TFBS matrix and selected TFs. We can see that some TFs have similar binding sites across the genomic regions.
-
-<!--, and that randomized lasso stability selection does reasonably well in selecting motifs from all given ranges of correlation, even highly correlated motifs. -->
-
-```{r TFBScor_selected}
+Next, we visualize the correlation structure of the TFBS matrix and selected TF motifs. While the colinearity of predictors is generally an issue in regression-based approaches where variables are selected, randomized lasso stability selection generally does better at selecting correlated predictors. In practice, we still see it select predictors with correlations as high as 0.9. However, it is good to keep in mind that this can be an issue, and that predictors that are extremely correlated with each other might not end up being selected. In a bad case example, if we have two identical predictors, neither of them would end up being selected as half of the times the regressions are done, one would be randomly selected, and the other one in the other half, giving each a selection probability of 0.5. If many cases of high correlations exist or this is a concern, one may consider selecting a representative set of predictors to use. 
```

Comment:
What about `neither of them could end up being selected if half of the` instead of `neither of them **would** end up being selected **as** half of the`

Reply:
Regarding the message about `NA` values, I have no strong opinion either way. I think it is not strictly needed, as this is documented behavior, but it also does not harm very much as it can be suppressed using `verbose = FALSE`.

 If it stays in, maybe one could set `verbose = TRUE` to one of the test cases so that it will not reduce our test coverage :-)